arkouda/strings.py

```diff
@@ -828,17 +828,114 @@ def save(self, prefix_path : str, dataset : str='strings_array',
         self.bytes.save(prefix_path=prefix_path, 
                                     dataset='{}/values'.format(dataset), mode=mode)
 
-    def register(self, user_defined_name : str) -> Strings:
+    def register(self, user_defined_name: str) -> Strings:
```

Comment:
add `@typechecked` to these methods and you don't have to manually check the types of the arguments.